src/packages/assets/misc/talk-20121004/index.html

```diff
@@ -132,7 +132,7 @@ <h1>sage-support mailing list a few days ago</h1>
 
 
 <div class="slide">
-<h1>Longterm Programing Goals (part 1)</h1>
+<h1>Longterm Programming Goals (part 1)</h1>
```

Comment:
wow, I didn't realize a talk from 2012 is still in the code. this starts to be of historic interest :-)

Reply:
Also "Programing" is a correct spelling of that word, which is more popular in American English.  

> "Long story short, either spelling of “programming” is acceptable and its popularity generally affects its trends in the English language. However, “programming” with two “m’s” is less likely to be docked by a Word processor set to American English." -- https://dictionarykiwi.com/2016/10/17/programming-versus-programing-which-is-preferred/